src/Tools/IdeCoreBenchmarks/NavigateToBenchmarks.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using AnalyzerRunner;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Engines;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.FindSymbols;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.MSBuild;
+using Microsoft.CodeAnalysis.NavigateTo;
+using Microsoft.CodeAnalysis.PatternMatching;
+using Microsoft.CodeAnalysis.SQLite.Interop;
+using Microsoft.CodeAnalysis.SQLite.v1;
+using Microsoft.CodeAnalysis.Storage;
+
+namespace IdeCoreBenchmarks
+{
+    [MemoryDiagnoser]
+    public class NavigateToBenchmarks
+    {
+        private readonly string _solutionPath;
+
+        private MSBuildWorkspace _workspace;
+
+        public NavigateToBenchmarks()
+        {
+            var roslynRoot = Environment.GetEnvironmentVariable(Program.RoslynRootPathEnvVariableName);
+            _solutionPath = Path.Combine(roslynRoot, @"C:\github\roslyn\Compilers.sln");
+
+            if (!File.Exists(_solutionPath))
+                throw new ArgumentException("Couldn't find Roslyn.sln");
+
+            Console.Write("Found roslyn.sln");
```

Comment:
Can we refactor a base class to do this rather than copy/pasting it all?

Reply:
So these are just helpers for me to quickly iterate and get numbers on.  I don't want to overengineer up front.  I think if/once these becoem things that run continuously that would make sense :)